### DISCUSSION TITLE
Break up vertical/horizontal stacks

### DISCUSSION THREAD

**Discussion by domoritz** [home-assistant/frontend#22701] [Dashboard] [answered]
I have a lot of vertical/horizontal stacks that could now be sections. I would love to have an easy way to break up a card into a section. Right now, I can't even access the code of a section to do it by hand so there isn't a good workaround.

**Comment by mariosker** [comment, marked as answer]
You could edit the code of the sections and copy paste the cards in the vertical/ horizontal stacks to the section

**Reply by domoritz** [reply]
Yes, I just discovered that you can get to the code with the three dots.
